Detections/OracleDatabaseAudit/OracleDBAuditNewIpForUser.yaml

```diff
@@ -0,0 +1,42 @@
+id: 39a0995e-f4a9-4869-a0ae-36d6d9049bfd
+name: OracleDBAudit - User connected to database from new IP
+description: |
+  'Detects when a user connects to database from new IP address.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: OracleDatabaseAudit
+    dataTypes:
+      - Syslog
+queryFrequency: 1h
+queryPeriod: 1h

```

Comment:
if you are going to lookback more than 1h in the query, this needs to be the lookback time, max of 14d

Reply:
updated